test/extended/placements/placements.go

```diff
@@ -0,0 +1,113 @@
+package placements
+
+import (
+	"os"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	apb "github.com/kcp-dev/kcp-tests/test/extended/apibinding"
+	nsc "github.com/kcp-dev/kcp-tests/test/extended/syncer"
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[area/transparent-multi-cluster]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-placements")
+	)
+
+	g.It("Author:knarra-Critical-[Regression] Verify default placements", func() {
+		// Shared compute could be only accessed from dev-provided test environments
+		// Skip for non-supported test environments
+		exutil.PreCheckEnvSupport(k, "kcp-stable.apps.kcp-internal", "kcp-unstable.apps.kcp-internal")
```

Comment:
If this is skipped, then this case is skipped prematurely even if your later BYO code can be run, right? How about reorder share compute snippets and BYO snippets? Or preferably use two sub-"g.It"s for them separately?

Reply:
Done, addressed.